book/src/drivers.md

```diff
@@ -116,7 +116,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [Tencent Cloud Object Storage](https://github.com/TencentCloud/kubernetes-csi-tencentcloud)| `com.tencent.cloud.csi.cosfs` | v1.0 | A Container Storage Interface (CSI) Driver for Tencent Cloud Object Storage | Persistent | Read/Write Multiple Pods | No |
 [TopoLVM](https://github.com/topolvm/topolvm)| `topolvm.io` | v1.1 | A Container Storage Interface (CSI) Driver for LVM | Persistent and Ephemeral | Read/Write Single Pod | Yes | Raw Block, Expansion, Topology Aware
 [TrueNAS](https://github.com/hpe-storage/truenas-csp) | `csi.hpe.com` | v1.3 | A community supported [Container Storage Provider](https://github.com/hpe-storage/container-storage-provider) (CSP) that leverages the HPE CSI Driver for Kubernetes. Works with TrueNAS CORE, TrueNAS SCALE and FreeNAS using iSCSI only | Persistent | Read/Write Multiple Pods | Yes | Raw Block, Snapshot, Expansion, Cloning
-[VAST Data](https://github.com/vast-data/vast-csi) | `csi.vastdata.com` | v1.0 | A Container Storage Interface (CSI) Driver for VAST Data | Persistent | Read/Write Multiple Pods | Yes |
+[VAST Data](https://github.com/vast-data/vast-csi) | `csi.vastdata.com` | v2.1 | A Container Storage Interface (CSI) Driver for VAST Data | Persistent and Ephemeral | Read/Write Multiple Pods | Yes | Snapshot, Expansion
```

Comment:
Column 3 is for CSI spec version, not driver version.  The latest CSI spec version is 1.7.0.

Reply:
Fixed in new commit